docs/presigned_urls.md

```diff
@@ -0,0 +1,354 @@
+│   │   └── example_sv_sample_1.bedpe
+│   ├── ···
+│   └── SAMPLE_N_ID/
+│       └── example_sv_sample_n.bedpe
+├── CNV_SUBDIR/                              # Contains CNV files (tsv)
+│   ├── SAMPLE_1_ID/
+│   │   └── example_cnv_sample_1.tsv
+│   ├── ···
+│   └── SAMPLE_N_ID/
+│       └── example_cnv_sample_n.tsv
+├── DRIVERS_SUBDIR/                          # Contains driver mutation files (tsv or json)
+│   ├── SAMPLE_1_ID/
+│   │   └── example_drivers_sample_1.tsv
+│   ├── ···
+│   └── SAMPLE_N_ID/
+│       └── example_drivers_sample_n.tsv
+├── SNV_SUBDIR/                              # Contains SNV files (vcf + tbi)
+│   ├── SAMPLE_1_ID/
+│   │   ├── example_snv_sample_1.vcf.gz
+│   │   └── example_snv_sample_1.tbi
+│   ├── ···
+│   └── SAMPLE_N_ID/
+│   │   ├── example_snv_sample_n.vcf.gz
+│   │   └── example_snv_sample_n.tbi
+├── INDEL_SUBDIR/                            # Contains indel files (vcf + tbi)
+│   ├── SAMPLE_1_ID/
+│   │   ├── example_indel_sample_1.vcf.gz
+│   │   └── example_indel_sample_1.tbi
+│   ├── ···
+│   └── SAMPLE_N_ID/
+│   │   ├── example_indel_sample_n.vcf.gz
+│   │   └── example_indel_sample_n.tbi
+├── READ_ALIGNMENTS_SUBDIR/                  # Contains read alignment files (bam + bai)
+│   ├── SAMPLE_1_ID/
+│   │   ├── example_reads_sample_1.bam
+│   │   └── example_reads_sample_1.bai
+│   ├── ···
+│   └── SAMPLE_N_ID/
+│   │    ├── example_reads_sample_n.bam
+│   │    └── example_reads_sample_n.bai
+└── CONFIGS_SUBDIR/                          # Contains timestamped configuration files (json)
+    ├── example_config_a.json
+    ├── example_config_b.json
+    ├── ···
+    └── example_config_z.json
+```
+
+?> The only categories that require subdirectories for *all* samples within the sample ID list are those for `SNV`s and `CNV`s. All other categories may contain some, all, or no sample subdirectories. If there is no sample data for a non-required category, that subdirectory need not be created.
+
+?> The `configuration files` subdirectory, if not provided, is by default identified by the name **"CONFIGS"**. If it does not exist within the bucket, a new directory with this name will be created, and generated configuration files added to it. The configuration files are named by the time of their creation.
+
+?> Note: If pairwise files don't pair correctly due to user error (e.g. a pair not of the same sample, not of correct file format), that category/parameter for that sample will not render within Chromoscope. This can be checked by manually inspecting samples within [cohort view](how-to-use.md#cohort-view).
+
+&nbsp;
+### Configuration File Creation Scripts: Usage
+
+The usage of the configuration file creation script via command line is as follows:
+
+```
+python3 generate_config_files.py --ids IDS --bucket BUCKET [--cohort COHORT] --cancer CANCER --assembly ASSEMBLY --sv SV --cnv CNV [--drivers DRIVERS] [--snv SNV] [--indel INDEL] [--reads READS] [--configs CONFIGS] [--expiration EXPIR]
```

Comment:
Maybe I missed it, but if not.. we should point the reader where the script lives in the repo